site/experimental-features.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Experimental features
```

Comment:
Let's instead have a proxy.md, and a short section in features.md - analogous to what we have for weaveDNS.